torch/csrc/distributed/c10d/NCCLUtils.hpp

```diff
@@ -128,6 +132,24 @@
     TORCH_CHECK_WITH(DistBackendError, false, err);                           \
   }
 
+#define C10D_SCHED_SLEEP()     \
+  std::this_thread::sleep_for( \
```

Comment:
Define an inline function instead of a macro?

Reply:
That's true in general. The special case here is that I am doing macro expansion below and passing function pointer does not seem very straightforward.